refarch-gateway/src/main/resources/application-local.yml

```
@@ -37,4 +37,16 @@ spring:
+
+logging:
+  include-application-group: false # disabled because application group is only relevant when deployed
+  structured:
+    format:
+      console: # set to empty to disable structured logging locally
+  level:
+    root: INFO
+    de.muenchen.refarch: DEBUG
+    org.springframework.web: DEBUG
+    org.springframework.cloud.gateway: TRACE
+    org.springframework.security: DEBUG
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)